<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btn_addImages">
          <property name="text">
           <string>Add Images By ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbl_age">
          <property name="text">
           <string>Age</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbl_name">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lbl_address">
          <property name="text">
           <string>Address</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QRadioButton" name="rdbtn_id">
          <property name="text">
           <string>&amp;ID</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" rowspan="2">
         <widget class="QLineEdit" name="lnedt_select"/>
        </item>
        <item row="5" column="3">
         <widget class="QRadioButton" name="rdbtn_name">
          <property name="text">
           <string>&amp;Name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" rowspan="2" colspan="2">
         <widget class="QPushButton" name="btn_select">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="btn_showImage">
          <property name="text">
           <string>Show Image By ID</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="2">
         <widget class="QLineEdit" name="lnedt_showImage"/>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QPushButton" name="btn_insert">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="lnedt_address"/>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="lnedt_age"/>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="lnedt_addImage"/>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="lnedt_name"/>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btn_delete">
          <property name="text">
           <string>Delete By ID</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QLineEdit" name="lnedt_delete"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lnedt_name</tabstop>
  <tabstop>lnedt_age</tabstop>
  <tabstop>lnedt_address</tabstop>
  <tabstop>btn_insert</tabstop>
  <tabstop>lnedt_addImage</tabstop>
  <tabstop>btn_addImages</tabstop>
  <tabstop>lnedt_select</tabstop>
  <tabstop>rdbtn_id</tabstop>
  <tabstop>rdbtn_name</tabstop>
  <tabstop>btn_select</tabstop>
  <tabstop>lnedt_delete</tabstop>
  <tabstop>btn_delete</tabstop>
  <tabstop>lnedt_showImage</tabstop>
  <tabstop>btn_showImage</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
